weblate/addons/models.py

```diff
@@ -42,25 +43,41 @@
 
 
 class AddonQuerySet(models.QuerySet):
-    def filter_component(self, component):
+    def filter_for_execution(self, component):
         return self.prefetch_related("event_set").filter(
-            (Q(component=component) & Q(project_scope=False))
-            | (Q(component__project=component.project) & Q(project_scope=True))
+            Q(component=component)
+            | Q(component__project=component.project)
```

Comment:
I think this is extra:

```suggestion
```

It would cause execution of other component add-ons on all components in a project.